src/components/KDropdown/KDropdownItem.vue

```diff
@@ -1,12 +1,12 @@
 <template>
   <li
     class="k-dropdown-item"
-    :class="{
+    :class="[{
       'has-divider': hasDivider,
       'disabled': disabled,
       'danger': danger || isDangerous,
-      'dropdown-selected-option': selected
-    }"
+      'dropdown-selected-option': selected,
+    }, $attrs.class, $attrs.style]"
```

Comment:
question: why are we moving all of the attributes? This is a larger breaking change